src/commands/buildCommands/trn.js

```diff
@@ -1,40 +1,60 @@
+const { promisify } = require('util');
```

Comment:
Is this a custom thing?

Reply:
nope, introduced in Node v8.something - just a simple way of converting callback-based async functions in Node (of which there are many) to a Promise-based interface to avoid callback hell

Reply:
Sweet!